src/modules/util.py

```diff
@@ -0,0 +1,14 @@
+
+def castInt(val):
+    try:
+        number = int(val)
+        return number
+    except ValueError:
+        return False
```

Comment:
Returning False for both of these seems like a bad idea.

Maybe catch the ValueError and raise a more descriptive error instead? Or log some debug out to the user and `exit(1)` early?

Reply:
Assertions are also valid to raise

`assert val.isnumeric(), "Sorry the value for XXXX field is non-numerical. Please enter a valid number."